frontend/simple-mercari-web/src/components/ItemDetail/ItemDescription.tsx

```diff
@@ -93,24 +119,19 @@ export const ItemDescription: React.FC<{ item: Item, isOwner: boolean}>  = ({ite
                         <Button variant="contained" onClick={onSubmit} id="buy-now-btn" color="primary" sx={{ mt: 3}}>
                           Buy now
                         </Button>
-                        {/**Also check here if in person passcode is enabled */}
-                        <FormControlLabel sx={{ mt: 3 }}
-                          control={
-                            <Checkbox
-                              checked={imWithSeller}
-                              onChange={(event) => setImWithSeller(event.target.checked)}
-                            />
-                          }
-                          label="I'm with the owner"
-                        />
-                        <TextField sx={{ mt: 2, ml: 3 }}
-                          id="inPersonKey"
-                          name="inPersonKey"
-                          //value={values.inPersonKey}
-                          //onChange={onValueChange}
-                          label="In Person Passcode"
-                          disabled={!imWithSeller}
-                        />
+                          {isInPersonAvailable && (
+                              <>
+                                  <FormControlLabel sx={{mt: 3}}
+                                                  control={<Checkbox
+                                                      checked={imWithSeller}
+                                                      onChange={(event) => setImWithSeller(event.target.checked)}/>}
+                                                  label="I'm with the owner"/><TextField sx={{mt: 2, ml: 3}}
+                                                                                         id="inPersonKey"
```

Comment:
your spacing is pretty off here but its not super important for running code

Reply:
Yeah I don't know why my editor likes that style lol
yeah, let me attach the demo so you can have a look before I merge this

Reply:
just saw the preview! it looks wonderful, thanks for your hardwork!